xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/WinVideoFilter.cpp

```diff
@@ -261,24 +261,22 @@ bool CYUV2RGBShader::Create(unsigned int sourceWidth, unsigned int sourceHeight,
 
   switch (fmt)
   {
-  case RENDER_FMT_YUV420P:
-  case RENDER_FMT_YUV420P10:
-  case RENDER_FMT_YUV420P16:
```

Comment:
thosee are not dropped: https://github.com/xbmc/xbmc/pull/11946/commits/320c5735499c1325cce004dc45279918ff2def2a